### ISSUE TITLE
Is it feasible to use DNC on own data?

### ISSUE THREAD

**Issue by saveli** [FlorianKrey/DNC#6]
The relevant parts of `run.sh` are removed (stages -1 to 3) and the used TDNN for the 32-dimensional d-vector embedding is unclear to me from the section 5.2 of the paper.
In issue #2 you mention any d-vector embedding could be used, but is this really true? Some parameters have to be identical, don't they (window size, overlap, sample rate,...)?
How would you have to edit the AMI-label files (the offsets probably)?
What files would I have to create for own data to use the decoder on these?

tldr: Is it feasible to use DNC on own data?
